signify-ts-test/src/vlei-issuance.ts

```diff
@@ -65,23 +64,53 @@ import {
   LE_INTERNAL_NAME,
   vleiServerUrl,
   witnessIds,
+  SCHEMAS,
+  RULES,
 } from "./constants";
 
-import { User, buildUserData } from "./utils/handle-json-config";
+import {
+  User,
+  CredentialInfo,
+  buildUserData,
+  buildCredentials,
+} from "./utils/handle-json-config";
+import { EcrTestData, buildTestData } from "./utils/generate-test-data";
+import fs from "fs";
+import path from "path";
 
 export class VleiIssuance {
-  secretsJsonPath: string = "./src/config/";
+  secretsJsonPath: string = "config/";
```

Comment:
eventually migrate away from concept/naming of 'secrets' as that is only a part of the configuration now.

Reply:
I'll do that in my next PR with iteration 2 changes for the multisig